src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticDF.scala

```diff
@@ -122,6 +122,8 @@ object ForestChangeDiagnosticDF extends SummaryDF {
     "protected_areas_area", // protectedAreasArea
     "peat_area", // peatlandsArea
     "arg_otbn_area", // argOTBNArea
+    "wdpa_area", // detailedProtectedAreas
```

Comment:
We already have fields using the term `protected_areas` to represent the less detailed wdpa (e.g. `protected_areas__area` 3 lines above), so I'd rename this to something that more clearly distinguishes it. Otherwise might be confusing in the future.

It seems like that field only ever looks at the total WDPA intersection and doesn't care about categories. So maybe this field can be protected_area_by_category or something? 

I'm also wondering if we need to read two protected areas layers or can just use the detailed protected areas for both fields. The total without categories should be the same for both, right?

Reply:
Thanks for the suggestions!

Renamed the new field to protected_areas_by_category and various internal variables/fields. to protectedAreasByCategories, etc.

Also, as you suggest, changed to only using the detailed categories layer to determine if an area is protected and for the detailed categories.  So, as you say, we only read the one layer now for both fields.